content/realtime/inband-occupancy.textile

```diff
@@ -0,0 +1,124 @@
+    - Occupancy channel parameter#occupancy-channel-parameter
+    - Occupancy events#occupancy-events
+    - See also#see-also
+---
+
+h2(#overview). Overview
+
+The inband channel occupancy feature allows a client to subscribe to occupancy metadata events relating to a single channel, and for those events to be delivered to the client as messages on the channel itself. Occupancy metadata events received as messages on the channel are known as __inband events__.
+
+The @Occupancy@ type is defined in the "Channel Metadata documentation":/realtime/channel-metadata#occupancy.
+
+h2(#occupancy-channel-parameter). Occupancy channel parameter
+
+Inband channel occupancy events are enabled in a channel attachment by specifying the @occupancy@ parameter. This is described in more detail in the "Channel Parameters documentation":/realtime/channel-params. The value of the parameter specifies the occupancy fields of interest, and the supported parameter values are:
+
+* @metrics@: this enables events containing the full @Occupancy@ details in their @data@ payload. Events are sent when the count for any of the included categories changes, up to a maximum rate of one message per second.
+* @metrics.<category>@:  this enables events whose @data@ payload contains an @Occupancy@ value containing the occupancy, that is the client count, for only the given category. Events are sent when the count for the given category changes, up to a maximum rate of one message per second.
+
+Channel parameters are a general mechanism for specifying attributes of a channel attachment. See the "Channel parameters documentation":/realtime/channel-params for further details.
+
+The following example shows how to specify the occupancy channel parameter:
+
+```[javascript]
+var channelOpts = { params: { occupancy: 'metrics' } };
+var channel = ably.channels.get('{{RANDOM_CHANNEL_NAME}}', channelOpts);
+```
+
+In this case all occupancy metrics are contained in occupancy messages.
+
+The following example shows how to specify a category:
+
+```[javascript]
+var channelOpts = { params: { occupancy: 'metrics.subscribers' } };
+var channel = ably.channels.get('{{RANDOM_CHANNEL_NAME}}', channelOpts);
+```
+
+In this case only occupancy metrics for @subscribers@ are contained in occupancy messages.
+
+h2(#occupancy-events). Occupancy events 
+
+Events in a channel that contain metadata have an event name qualified with the @[meta]prefix@. For example, occupancy messages have an event name of @[meta]occupancy@. Messages published by clients do not contain this metadata. The following code snippet illustrates this:
+
+```[javascript]
+var channelOpts = { params: { occupancy: 'metrics' } };
+var channel = ably.channels.get('{{RANDOM_CHANNEL_NAME}}', channelOpts);
+
+channel.subscribe('[meta]occupancy', (message) => {
+  console.log('inband: ', message.data);
+});
+```
+
+In this example only occupancy messages will be logged to the console, not user-specified messages.
```

Comment:
> user-specified messages.

"user-published messages" is probably closer. 'specified' is ambiguous - after all, the user has specified they want to subscribe to occupancy updates

Reply:
Fixed.